src/app/client/src/app/modules/dashboard/components/report/report.component.spec.ts

```diff
@@ -375,6 +375,53 @@ describe('ReportComponent', () => {
     component['addSummaryBtnClickStream$'].next({ title: 'Add report Summary', type: 'report' });
   });
 
+  it('should get all report data', fakeAsync(() => {
+    component.ngOnInit();
+    tick(1000);
+
+    component.reportData = {
+      charts:[{
+        chartData:chartData
+      }
+    ]
+    }
+    const data = component.getAllChartData();
+    expect(data).toEqual(chartData);
+  }));
+
+  it('should change the filter', fakeAsync(() => {
+    component.ngOnInit();
+    tick(1000);
+    const data = component.filterChanged({
+      chartData:chartData,
+      filters:filters
+    });
+  }));
+  it('should get chartData', fakeAsync(() => {
```

Comment:
no assertions in this test case block... then it's of no use right. Please add relevant assertions

Reply:
resolved